io_export_paper_model.py

```diff
@@ -1226,47 +1320,40 @@ def __init__(self, uvedge, default_width, index, other: UVEdge):
         len_a = min(len_a, (edge.length * sin_b) / (sin_a * cos_b + sin_b * cos_a))
         len_b = 0 if sin_b == 0 else min(sticker_width / sin_b, (edge.length - len_a * cos_a) / cos_b)
 
-        sin, cos = edge.y / edge.length, edge.x / edge.length
+        tangent = edge.normalized()
+        cos, sin = tangent
```

Comment:
if possible to leave type ambiguity out, e.g., let's use style to type python more strongly than python is typed

Reply:
make this type explicit